### ISSUE TITLE
Training logs

### ISSUE THREAD

**Issue by yuanpengtu** [bytedance/fc-clip#6]
Hi, thanks for your great work. Could you provide the training log on the COCO dataset please? I'd like to compare my reproduction results to find out what went wrong. Thanks.

**Comment by Jeff001128** [comment]
Hi, I'm also reproducing the results. Can you show your training logs here? Here is mine:
![image](https://github.com/bytedance/fc-clip/assets/77483744/e52e0aee-66e6-483d-bb2b-c11396921d5a)
I use two GPUs with batch size=4 for training, and I'm unsure if the small batch size leads to the oscillation of loss. Is yours the same?

Thanks.

**Comment by wusize** [comment]
Cannot reproduce the results of convnext on ADE20k. Has anybody successfully reproduce the result?

**Comment by cornettoyu** [comment]
Hi,

Thanks for your interest. Could you please specify which results are wrong in your reproduction (e.g., sharing your experiment settings, results so we can look into the problem)? We will also upload the training log soon.

**Comment by cornettoyu** [comment]
> Hi, thanks for your great work. Could you provide the training log on the COCO dataset please? I'd like to compare my reproduction results to find out what went wrong. Thanks.

Hi,

Thanks for your interest. Could you please specify which results are wrong in your reproduction (e.g., sharing your experiment settings, results so we can look into the problem)? We will also upload the training log soon.

**Comment by cornettoyu** [comment]
> Hi, I'm also reproducing the results. Can you show your training logs here? Here is mine: ![image](https://user-images.githubusercontent.com/77483744/274715496-e52e0aee-66e6-483d-bb2b-c11396921d5a.png) I use two GPUs with batch size=4 for training, and I'm unsure if the small batch size leads to the oscillation of loss. Is yours the same?
> 
> Thanks.

Hi,

We have not tried using a different setting. Please ensure you are following the provided config if aiming at reproducing our results. Thanks.

**Comment by cornettoyu** [comment]
> Cannot reproduce the results of convnext on ADE20k. Has anybody successfully reproduce the result?

Can you provide your results here so we can look into what is wrong? We will upload our training log as well soon. It is hard to tell what is wrong with no details except "cannot reproduce" :)

**Comment by cornettoyu** [comment]
> > > Cannot reproduce the results of convnext on ADE20k. Has anybody successfully reproduce the result?
> > 
> > 
> > Can you provide your results here so we can look into what is wrong? We will upload our training log as well soon. It is hard to tell what is wrong with no details except "cannot reproduce" :)
> 
> Hi! Thank you for your reply. I will try another run of the training on convnext and share you the log. By the way, did you save the checkpoint of the best result or simply the last one. I found the PQ on ADE saturated very quickly. And further training it for more iterations only improves the performance on COCO.

Thanks for checking. Yes, the PQ can achieves 25~26 at the first 50k steps, while improvement can be relatively smaller in the remaining 300k steps. We keep the best checkpoint for our final results.

**Comment by cornettoyu** [comment]
Please see the attached
[log file](https://github.com/bytedance/fc-clip/files/13231285/metrics.json) for a reference of training log FC-CLIP with ConvNeXt-L. The provided checkpoint is at step 309999, with "panoptic_seg/PQ": 26.788164947280208

Furthermore, we ran the experiments with CUDA 11.3, PyTorch 1.21.1, NVIDIA Tesla V100-SXM2-32GB, hope these information may help you figure out the problem. And free feel to reach out if you encounter any problem with details :)

**Comment by yxchng** [comment]
@cornettoyu how do you select the best checkpoint? do you evaluate all of them?

**Comment by yxchng** [comment]
@wusize what results do you get?

**Comment by SuleBai** [comment]
> Please see the attached [log file](https://github.com/bytedance/fc-clip/files/13231285/metrics.json) for a reference of training/validation log FC-CLIP with ConvNeXt-L. The provided checkpoint is at step 309999, with "panoptic_seg/PQ": 26.788164947280208
> 
> Furthermore, we ran the experiments with CUDA 11.3, PyTorch 1.12.1, NVIDIA Tesla V100-SXM2-32GB, hope these information may help you figure out the problem. And free feel to reach out if you encounter any problem with details :)

Hi! It seems that PyTorch 1.12.1 is **inconsistent with** the one you provided in the `INSTALL.md`. So if I want to reproduce your results, should I run the command below when preparing the environment?
`conda install pytorch==1.12.1 torchvision==0.13.1 torchaudio==0.12.1 cudatoolkit=11.3 -c pytorch -c nvidia`
